builder/dockerfile/dispatchers.go

```diff
@@ -212,7 +212,7 @@ func from(b *Builder, args []string, attributes map[string]bool, original string
 	} else {
 		// TODO: don't use `name`, instead resolve it to a digest
 		if !b.options.PullParent {
-			image, err = b.docker.GetImageOnBuild(name)
+			image, _ = b.docker.GetImageOnBuild(name)
```

Comment:
In this case we can move the declaration of `err` below where it's actually used.